ui-manchette/src/styles/manchette.css

```diff
@@ -9,20 +15,34 @@
     width: 100%;
     backdrop-filter: blur(8px);
     -webkit-backdrop-filter: blur(8px);
-    border-top: solid 1px;
-    border-right: solid 1px;
-    @apply border-black-25;
-    border-radius: 0px 0px 0px 10px;
 
-    .zoom-out:disabled {
-      opacity: 0.4;
-    }
 
-    .zoom-in:disabled {
-      opacity: 0.4;
+    .zoom-buttons {
+      display: flex;
+      align-items: center;
+      height: 100%;
+
+      button {
+        border-right: solid 1px #d3d1cf;
+        border-top: solid 1px white;
+        height: 100%;
+        padding-inline: 12px;
+      }
+
+      button:disabled {
+        span {
+          color: #B6B2AF
+        }
+      }
+
+      .zoom-reset {
+        color: #797671;
```

Comment:
Could we use the design system's colors instead of hardcoded hex codes? Something like `@apply text-grey-50`? (Ditto for other colors)